hirola/front/views.py

```diff
@@ -104,9 +104,11 @@ def phone_profile_view(request, phone_id):
     colors = PhonesColor.objects.filter(phone=phone_id, is_in_stock=True)
     if not phone:
         return redirect("/error")
+    (phone_categories, social_media) = various_caches()
```

Comment:
For all the view methods, also pass the social_media variable just as you passed the phone_categories variable.😔I forgot to include it.